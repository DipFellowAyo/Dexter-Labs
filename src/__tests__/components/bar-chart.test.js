import React from 'react'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'

import BarChart from '../../../src/components/bar-chart'

describe('BarChart', () => {
  let barChart

  beforeEach(() => {
    barChart = shallow(<BarChart />)
  })

  it('renders correctly', () => {
    expect(toJson(barChart)).toMatchSnapshot()
  })
})
