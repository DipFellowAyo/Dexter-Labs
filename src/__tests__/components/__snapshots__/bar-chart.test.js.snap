// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BarChart renders correctly 1`] = `
<ForwardRef
  data={[Function]}
  options={
    Object {
      "maintainAspectRatio": true,
      "plugins": Object {
        "legend": false,
      },
      "responsive": true,
      "scales": Object {
        "xAxis": Object {
          "grid": Object {
            "display": false,
          },
        },
        "y": Object {
          "max": 1000,
          "min": 100,
        },
      },
    }
  }
/>
`;
